src/styles.css

```
@@ -417,19 +466,34 @@ footer a:hover{text-decoration:underline}
 @keyframes shimmer{0%{background-position:200% 0;}100%{background-position:-200% 0;}}
 @media(prefers-reduced-motion:reduce){.skeleton-logo,.skeleton-line,.skeleton-body,.skeleton-pill{animation:none}}
 
-/* ── TABLET ── */
+/* ── SCROLL TOP BUTTON ── */
+.scroll-top-btn{position:fixed;bottom:28px;right:24px;width:40px;height:40px;border-radius:50%;background:var(--orange);color:white;border:none;font-size:18px;cursor:pointer;display:none;z-index:50;box-shadow:0 3px 10px rgba(244,107,14,.35);transition:opacity .2s}
```

Stylelint (17.11.1)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)